AudioSinkExample/ViewController.swift

```diff
@@ -0,0 +1,692 @@
+                                                             resultHandler: { (result, error) in
+                                                                if let validResult = result {
+                                                                    self.speechLabel?.text = validResult.bestTranscription.formattedString
+                                                                } else if let error = error {
+                                                                    self.speechLabel?.text = error.localizedDescription
+                                                                    // TODO: CE - Stop recognition here, or should it be done in the recognizer?
+                                                                }
+
+                                                                UIView.animate(withDuration: 0.1, animations: {
+                                                                    self.view.setNeedsLayout()
+                                                                    self.view.layoutIfNeeded()
+                                                                })
+        })
+    }
+
+    func prepareLocalMedia() {
+
+        // Create an audio track.
+        localAudioTrack = TVILocalAudioTrack.init()
+
+        // Create a video track which captures from the front camera.
+        if (TVICameraCapturer.isSourceAvailable(TVICameraCaptureSource.frontCamera)) {
+
+            // We will render the camera using TVICameraPreviewView.
+            camera = TVICameraCapturer(source: .frontCamera, delegate: self, enablePreview: true)
+            localVideoTrack = TVILocalVideoTrack.init(capturer: camera!)
+
+            if (localVideoTrack == nil) {
+                logMessage(messageText: "Failed to create video track!")
+            } else {
+                logMessage(messageText: "Video track created.")
+
+                if let preview = camera?.previewView {
+                    let tap = UITapGestureRecognizer(target: self, action: #selector(ViewController.recognizeLocalAudio))
+                    preview.addGestureRecognizer(tap)
+                    view.addSubview(preview);
+                }
+            }
+        } else if (localAudioTrack != nil) {
+            logMessage(messageText: "Front camera is not available, using microphone only.")
+        } else {
+            logMessage(messageText: "Failed to create audio track!")
+        }
+    }
+
+    func setupRemoteVideoView(publication: TVIRemoteVideoTrackPublication) {
+        // Create a `TVIVideoView` programmatically, and add to our `UIStackView`
+        if let remoteView = TVIVideoView.init(frame: CGRect.zero, delegate:nil) {
+            // We will bet that a hash collision between two unique SIDs is very rare.
+            remoteView.tag = publication.trackSid.hashValue
+
+            // `TVIVideoView` supports scaleToFill, scaleAspectFill and scaleAspectFit
```

Comment:
very minor - grammar - period after the first line.
`...  scaleAspectFit.`

Reply:
I believe this was copy/pasted from another example.